superset/explore/form_data/commands/entry.py

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing_extensions import TypedDict
+
+
+class Entry(TypedDict):
+    owner: int
+    dataset_id: int
+    chart_id: int
```

Comment:
Shouldn't `dataset_id` and `chart_id` be `Optional[int]`?

Reply:
The `dataset_id` is always required but the `chart_id` should be optional. Will change.